File: app/src/main/java/hedera/hgc/hgcwallet/app_intent/IntentParams.java
package hedera.hgc.hgcwallet.app_intent;

import android.support.annotation.NonNull;

import hedera.hgc.hgcwallet.BuildConfig;

public interface IntentParams {
    public static final @NonNull String APP_URL_SCHEMA = "https";
    public static final @NonNull String APP_HOST = BuildConfig.USE_TEST_BRANCHIO ? "hedera.test-app.link" : "hedera.app.link";
    public static final @NonNull String APP_URL_PATH = BuildConfig.USE_TEST_BRANCHIO ? "OYEncK9sLQ" : "5vuEEQhtLQ";
}

File: app/src/main/java/hedera/hgc/hgcwallet/app_intent/IntentParams.kt
package hedera.hgc.hgcwallet.app_intent

import hedera.hgc.hgcwallet.BuildConfig

interface IntentParams {
    companion object {
        val APP_URL_SCHEMA = "https"
        val APP_HOST = if (BuildConfig.USE_TEST_BRANCHIO) "hedera.test-app.link" else "hedera.app.link"
        val APP_URL_PATH = if (BuildConfig.USE_TEST_BRANCHIO) "OYEncK9sLQ" else "5vuEEQhtLQ"
    }
}
